### PR TITLE
LPD-37003  | Test Fix | Poshi >>> Playwright

### PR DESCRIPTION
**Pull request by rafaellajordao**
### Motivation:
Test failing due to Poshi limitations.

### Proposed Solution:
Transcribe test to playwright in order to facilitate maintenance.

**More details:** 
[LPD-37002](https://liferay.atlassian.net/browse/LPD-37002)
[LPD-37003](https://liferay.atlassian.net/browse/LPD-37003)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts [hunk 1]
```diff
@@ -347,6 +348,96 @@ test(
...
 );
 
+test(
+	'Assert the pages list shows a list of pages',
+	{
+		tag: '@Legacy',
+	},
+
+	async ({apiHelpers, page}) => {
+		const pageTitle1 = 'My Page';
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/page.spec.ts:358]: @rafaellajordao, I think `pageTitle1` should be closer its usage. Also, as it is referenced after `pageTitle2` and `pageTitle3` maybe we should consider renaming all three variables according to the order.

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts [hunk 2]
```diff
@@ -1109,3 +1109,66 @@ test(
...
+			await changeTimeFilter({
+				page,
+				timeFilterPeriod: 'Last 24 hours',
+			});
+		});
+
+		await test.step('Assert the page list', async () => {
+			await page.getByPlaceholder('Search').first().click();
+			await page.getByPlaceholder('Search').first().fill(href);
+			await page.getByPlaceholder('Search').first().press('Enter');
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/page.spec.ts:1167]: @rafaellajordao as `page.getByPlaceholder('Search')` is being repeated, maybe we can extract that to a `const`

**Review by marcosapmf — CHANGES_REQUESTED** [review]
Thanks @rafaellajordao!
Just minor adjustments. Please see my comments

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts (hunk 2 above)
Comment (rafaellajordao) [reply modules/test/playwright/tests/osb-faro-web/page.spec.ts]: fixed!

**Comment by rafaellajordao** [comment]
> ### ❌ ci:test:analytics-cloud - 237 out of 255 jobs passed in 2 hours 48 minutes
> Click here for more details.

all the tests modified in this PR are passing normally! :smile:

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
> ### ❌ ci:test:analytics-cloud - 241 out of 255 jobs passed in 3 hours 34 minutes
> Click here for more details.

all the tests modified in this PR are passing correctly!

**Review by marcosapmf — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**